2021-09-22 21:46:36,303 INFO    MainThread:107116 [wandb_setup.py:_flush():69] setting env: {}
2021-09-22 21:46:36,303 INFO    MainThread:107116 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-22 21:46:36,304 INFO    MainThread:107116 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20210922_214636-25dkuazo/logs/debug.log
2021-09-22 21:46:36,304 INFO    MainThread:107116 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20210922_214636-25dkuazo/logs/debug-internal.log
2021-09-22 21:46:36,304 INFO    MainThread:107116 [wandb_init.py:init():381] calling init triggers
2021-09-22 21:46:36,304 INFO    MainThread:107116 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-22 21:46:36,304 INFO    MainThread:107116 [wandb_init.py:init():430] starting backend
2021-09-22 21:46:36,304 INFO    MainThread:107116 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-22 21:46:36,311 INFO    MainThread:107116 [backend.py:ensure_launched():135] starting backend process...
2021-09-22 21:46:36,393 INFO    MainThread:107116 [backend.py:ensure_launched():140] started backend process with pid: 107649
2021-09-22 21:46:36,395 INFO    MainThread:107116 [wandb_init.py:init():435] backend started and connected
2021-09-22 21:46:36,397 INFO    MainThread:107116 [wandb_init.py:init():494] updated telemetry
2021-09-22 21:46:36,398 INFO    MainThread:107116 [wandb_init.py:init():517] communicating current version
2021-09-22 21:46:37,100 INFO    MainThread:107116 [wandb_init.py:init():522] got version response 
2021-09-22 21:46:37,100 INFO    MainThread:107116 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-22 21:46:37,234 INFO    MainThread:107116 [wandb_init.py:init():557] starting run threads in backend
2021-09-22 21:46:41,528 INFO    MainThread:107116 [wandb_run.py:_console_start():1605] atexit reg
2021-09-22 21:46:41,529 INFO    MainThread:107116 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-22 21:46:41,529 INFO    MainThread:107116 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-22 21:46:41,535 INFO    MainThread:107116 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-22 21:46:41,535 INFO    MainThread:107116 [wandb_init.py:init():582] run started, returning control to user process
2021-09-22 21:47:43,904 INFO    MainThread:107116 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 255
2021-09-22 21:47:43,905 INFO    MainThread:107116 [wandb_run.py:_restore():1547] restore
2021-09-22 21:47:46,513 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-22 21:47:46,805 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-22 21:47:46,922 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-22 21:47:47,304 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 21894
}

2021-09-22 21:47:47,407 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 21894
}

2021-09-22 21:47:47,510 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21894
  total_bytes: 21894
}

2021-09-22 21:47:47,613 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21894
  total_bytes: 21894
}

2021-09-22 21:47:47,715 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21894
  total_bytes: 21894
}

2021-09-22 21:47:47,818 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21894
  total_bytes: 21894
}

2021-09-22 21:47:48,073 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21894
  total_bytes: 21894
}

2021-09-22 21:47:48,342 INFO    MainThread:107116 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 21894
  total_bytes: 21894
}
local_info {
}

2021-09-22 21:47:49,610 INFO    MainThread:107116 [wandb_run.py:_show_summary():1853] rendering summary
2021-09-22 21:47:49,610 INFO    MainThread:107116 [wandb_run.py:_show_history():1898] rendering history
2021-09-22 21:47:49,611 INFO    MainThread:107116 [wandb_run.py:_show_files():1947] logging synced files
